Chrysalis-Papilio/Soen490-Papilio-Website
# PAPILIO-41-story

## General 

A company should be able to subscribe to a level of ad tier and connect itself to the database to get that information back. It will allow them to create ads for the application. This information will be view in the billing section of the application.

_Issue number_: Papilio-41

## Task Description
- Connect the backend to the Payment form to create the adTier for the company
- Fetch the AdTier to view that information in Billing
- Create a button to unsubscribe to that ad tier in the dashboard
- Fix broken test
- Add Hotter

## Manual Test
1. Start the backend `npm run build && npm start`
2. Start the frontend `npm start`
3. Login in the application
4. Visit the add section 
5. Select one tier
6. Add the information for the credit card and press pay (You don't have to enter real information)
7.  You should be in the billing page
8. Make sure that the correct information is displayed 
9. Unsubscribe to the ad tier
10. No more information should be display

## Test
`npm test`
